draft-ietf-oauth-v2-1.md

```diff
@@ -2283,56 +2305,24 @@ audience), typically a single resource server (or a list of resource
 servers), in the token.  Restricting the use of the token to a
 specific scope is also RECOMMENDED.
 
-To protect against access token disclosure, confidentiality protection MUST
-be applied using TLS with a ciphersuite that provides
-confidentiality and integrity protection.  This requires that the
-communication interaction between the client and the authorization
-server, as well as the interaction between the client and the
-resource server, utilize confidentiality and integrity protection.
-Since TLS is mandatory to implement and to use with this
-specification, it is the preferred approach for preventing token
-disclosure via the communication channel.  For those cases where the
-client is prevented from observing the contents of the access token, token
-encryption MUST be applied in addition to the usage of TLS
-protection.  As a further defense against token disclosure, the
```

Comment:
@aaronpk TLS / channel-encryption is already a MUST. This is probably redundant.